src/Umbraco.Core/Persistence/Repositories/MediaRepository.cs

```diff
@@ -612,13 +506,111 @@ private string EnsureUniqueNodeName(int parentId, string nodeName, int id = 0)
 
                     if (dto.Text.ToLowerInvariant().Equals(currentName.ToLowerInvariant()))
                     {
-                        currentName = nodeName + string.Format(" ({0})", uniqueNumber);
+                        currentName = nodeName + $" ({uniqueNumber})";

```

Comment:
Shouldn't this just use full string interpolation? `$"{nodeName} ({uniqueNumber})"`